components/IdeasMoveArea/index.js

```diff
@@ -0,0 +1,33 @@
+export default function IdeasMoveArea({
+  projects,
+  selectedProjectForIdea,
+  handleSelectedProjectForIdea,
+  handleAddIdeaToProject,
+}) {
+  return (
+    <article>
+      <p>Welchem Projekt möchtest du die Idee zuordnen?</p>
+      <ul>
+        {projects.map((project) => (
+          <li
+            key={project.id}
+            onClick={() => handleSelectedProjectForIdea(project.id)}
+          >
+            <article className="moveArea">
+              {selectedProjectForIdea === project.id && (
+                <span className="selected">-X|</span>
+              )}
+              <p>{project.title}</p>
+              {selectedProjectForIdea === project.id && (
+                <span className="selected">|X-</span>
+              )}
+            </article>
+          </li>
+        ))}
```

Comment:
The `onClick` should be placed on a `button` element because the `li` is not clickable by default (it's an a11y issue: the screen reader will not tell the user that the following content is clickable):
```suggestion
          <li
            key={project.id}
          >
          <button type="button" onClick={() => handleSelectedProjectForIdea(project.id)} >
            <article className="moveArea">
              {selectedProjectForIdea === project.id && (
                <span className="selected">-X|</span>
              )}
              <p>{project.title}</p>
              {selectedProjectForIdea === project.id && (
                <span className="selected">|X-</span>
              )}
            </article>
            </button>
          </li>
        ))}
```